openhands/runtime/utils/runtime_templates/Dockerfile.j2

```diff
@@ -105,28 +115,22 @@ RUN rm -rf ${OPENVSCODE_SERVER_ROOT}/extensions/{handlebars,pug,json,diff,grunt,
 # Install all dependencies
 WORKDIR /openhands/code
 
```

Comment:
This is easier to review but using split diff view, which can be found by changing `/files` to `/files?diff=split` in the URL